index.html

```diff
@@ -13,7 +13,7 @@
     <div class="tonic">
 
       <div>
-        <img src="images/tonic.png" alt=" Best Deal Ever" style= "width:520px;height:448px">
+        <img src="images/tonic.png" alt=" Best Deal Ever" width= "520" height= "448">
       </div>
 
       <div class="tonic-b">
```

Comment:
Please follow the design for the text section of the projects. Kindly use the same font and color from the design
![text-section](https://user-images.githubusercontent.com/35504149/134784648-0621b735-1503-4827-a26a-4eefceabf1eb.png)
.